src/reducers/login.js

```diff
@@ -0,0 +1,29 @@
+import { userConstants } from '../constants/userConstants';
+
+const user = JSON.parse(localStorage.getItem("user"));
+const initialState = user ? { loggedIn: true, ...user } : {};
+
+export function login(state = initialState, action) {
+    switch (action.type) {
+        case userConstants.LOGIN_REQUEST:
+            return {
+                loggingIn: true,
```

Comment:
You are using 'type' field in alert reducer, but do not use it here. Why? You should write your code in the uniform style.